app/assets/javascripts/modules/FirmMap.js

```diff
@@ -103,16 +103,34 @@ define(['jquery', 'DoughBaseComponent'],
   FirmMapProto._positionMarkers = function(gMap) {
     var $elements = this.$find('[data-dough-map-point]');
 
+    var infoWindow = new google.maps.InfoWindow();
+
     $elements.each($.proxy(function(_, element) {
       var $element = $(element),
           markerConfig = this._generateMarkerConfig($element);
 
       markerConfig.map = gMap;
-      new google.maps.Marker(markerConfig);
+      var marker = new google.maps.Marker(markerConfig);
```

Comment:
Can we just  shift this variable declaration (and the one on 106) into their respective groups within the blocks for style?